src/test/java/scenarios/webHomeworkTest.java

```diff
@@ -0,0 +1,54 @@
+package scenarios;
+
+import static pageObjects.WebPages.GooglePage.googleURL;
+
+import java.util.Locale;
+import org.openqa.selenium.Keys;
+import org.testng.Assert;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+import pageObjects.WebPages.GooglePage;
+import setup.BaseTest;
+
+public class webHomeworkTest extends BaseTest {
+
+    String searchText = "EPAM";
+
+    //test will be started with other "web" tests
+    @Parameters({"platformName"})
+    @Test(groups = {"web"}, description = "Check search results for the 'EPAM' word")
+    public void WebTest(String platformName) throws Exception {
+
+        //load google search page
+        getDriver().get(googleURL);
+        waitUntilPageLoad();
+
+        //create page object
+        GooglePage page = new GooglePage(getDriver());
+
+        System.out.println("Current platform: " + platformName);
+
+        //Agree with cookies policy for Hungary.
+        // If you have no this popup please commit.
+        acceptHungaryCookie(page);
+
+        //Click enter
+        switch (platformName.toUpperCase(Locale.ROOT)) {
+            case "ANDROID":
+                //Fill search field
+                page.fillSearchField(searchText);
+                getDriver().getKeyboard().pressKey(Keys.ENTER); //don't work for the IOs
+                break;
+            case "IOS":
+                page.getSearchField().click();
+                page.getSearchField().sendKeys(searchText + Keys.ENTER); //tried + "\n" as well
+                break;
+            default:
+                throw new Exception("Can't click Enter on " + platformName);
+        }
```

Comment:
It's worth implementing these logic on page layer, as the behavior is not related to test = business logic of an app

Reply:
Fixed